### PR TITLE
RA-Balance changes for the next playtest

### PR DESCRIPTION
**Pull request by abcdefg30**
Next try after #9623 was closed.

### CODE REVIEW

**Comment by MicroJOo** [comment]
also the flaktrucks require further tweaking ... they have been reported "extremely" usefull against "Hinds" ... this values here , shows why this is a "real issue"
_____________________________________
FLAKTRUCK:
        RevealsShroud:
	Range: 6c0
Weapon:
        FLAK-23:
	Range: 8c0
_____________________________________
HIND:
        RevealsShroud:
	Range: 10c0
Weapon:
        ChainGun:
	Range: 5c0

i can see what @ScottNZ did here ... the idea in theory was to make Flaktrucks work together with other (scout) units ... to make it more effective
but experiencing this on the battlefield is quite sobering , flaks are agile and fast ... and quickly to produce with the cost of only 600 (even a light tank costs 700) ... apart from that , its possible to "curtain" them ... this is what soviet-players do when they think its "time to clean up" on the front ... destroying artys/v2s and all aircrafts ....

i suggest to here to start fixing this with : 

Weapon:
        FLAK-23:
	Range: 8c0 -> 6c0

and lets see how this will turn out in playtests

**Comment by r34ch** [comment]
You could also buff the light tank to be a flak trucks direct counter. Lower the cost to $600 and buff the RoT since the speed for both are already the same. Perhaps buff the damage against 'Heavy' targets to also make them useful for early OT harassment and mid game hit and run.

**Comment by MicroJOo** [comment]
thats not gonna work ... in order to be a direct counter , we have to buff also the range of the ligh-tanks ...
there is allready plans to change the "ROT" of mediums and ligh-tanks ... but we should wait for this until we have  something to "trade" in , otherwise people could get mad 

better we keep it simple , and just see what the decreased flaktruck-range will do

**Comment by abcdefg30** [comment]
Added the `FLAK-23` range decrease.

**Comment by abcdefg30** [comment]
Updated.

**Comment by r34ch** [comment]
@MicroJOo I don't see why the light tank range would need increased if the flak truck range is being decreased? If that happened along with the RoT and price changes then the light tank would dominate the flak truck with its much heavier armour, far higher DPM and equal speed.

But anyway, like you say lets test the flak truck change first. I think it will be a good change but I'm curious about defending sub pens from air...

**Comment by MicroJOo** [comment]
r34ch : that you cant really defend a subpen from air-units hasnt been changed ... even when having FLAK-23 on Range: 8c0 .. it cant reach a hind from a straight coast ... but thats a different story , we all know something has to be changed on the naval stuff

about the light tank ... i dont see why we should give him the same speed as the flaktruck ...its still a "tank" ... and has other advantages like "leaving husks" , "crushing infantry" ... you can do some nice harrassement on powerplants and so on ... 

as said , ill up the ROT ... cause thats one thing that would actually make sense ... (its hard to explain though , why out of the sudden all the allies tanks get a buff ... so im for waiting on this

**Comment by reaperrr** [comment]
wrt the flak truck, if the range nerf turns out to be a little too large vs. aircraft, a possibility would be to split the weapon into AA-only and AG-only and then only re-increase range of the AA variant.

File: mods/ra/rules/infantry.yaml [hunk 1]
```diff
@@ -450,6 +450,14 @@ HIJACKER:
 	WithInfantryBody:
 		-IdleSequences:
 		StandSequences: stand
+	Crushable:
+		WarnProbability: 95
+	Cloak:
+		InitialDelay: 250
+		CloakDelay: 120
+		UncloakOnMove: true
+		CloakTypes: Cloak, Hijacker
```
Comment (obrakmann) [inline mods/ra/rules/infantry.yaml]: Why isn't just `Hijacker` not enough?

File: mods/ra/rules/misc.yaml [hunk 2]
```diff
@@ -126,10 +126,6 @@ CAMERA:
 	RevealsShroud:
 		Range: 10c0
 		Type: CenterPosition
-	ProximityCaptor:
-		Types: Camera
-	DetectCloaked:
-		Range: 10c0
```
Comment (obrakmann) [inline mods/ra/rules/misc.yaml]: This was actually on purpose, see here: fc2c46a2c25aad0c5c05a869d18e37762f673fe8

But cameras are used in too many places now for this to not potentially have bad side effects.

**Comment by obrakmann** [comment]
Confirmed that hijackers can still be crushed by harvesters :p

I hope we'll get some feedback on how this turns out during the playtest. This is quite a buff for the hijacker. Also not sure how the weapon range decrease for the flak truck will really affect things. But yeah, they are annoying things.

Oh well. :+1:, pending an answer on the `CloakTypes` question.

**Comment by MicroJOo** [comment]
@obrakmann : about your CloakTypes questio 

i would like to have also "base defenses" detect him ... and not rely completely on infantry to detect him , thats why "cloak" too , beside "hijacker"

considering i didnt up the price of the unit , this should be fair enough

**Comment by GraionDilach** [comment]
Something is definitely off here though, in the original PR of @MicroJOo IIRC there were stuff which could only detect the hijacker but now there isn't. `^Soldier` or `^Defense` should get a `DetectCloaked: Hijacker` then.

**Comment by abcdefg30** [comment]
> `^Soldier` or `^Defense` should get a `DetectCloaked: Hijacker` then.

That is `^Soldier`:
https://github.com/OpenRA/OpenRA/pull/9668/files#diff-5b0780203fcfc47fcb88e0b8d6d306b9R271

**Comment by GraionDilach** [comment]
Ah, tight, ignore me being stupid.

File: mods/ra/rules/misc.yaml (hunk 2 above)
Comment (abcdefg30) [inline mods/ra/rules/misc.yaml]: Added a new `camera.spyplane`.

**Comment by abcdefg30** [comment]
Updated.

**Comment by MicroJOo** [comment]
:+1:

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/629e1b74f2de01fe3ba3a104225402f91d915660

**Comment by r34ch** [comment]
Well fwiw I think the chinook sight range and hijacker buffs are good. 

I can't comment fully on the flak changes because I didn't really play many naval maps but the changes don't seem to break any of the main strategies on land with them.

**Pull request merged**